src/main/java/tpietzsch/example2/MultiVolumeShaderMip.java

```diff
@@ -29,16 +29,15 @@
 package tpietzsch.example2;
 
 import bdv.tools.brightness.ConverterSetup;
+
+import java.lang.Math;
 import java.util.AbstractMap;
 import java.util.Arrays;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.function.BiConsumer;
 import net.imglib2.type.numeric.ARGBType;
-import org.joml.Matrix4f;
-import org.joml.Matrix4fc;
-import org.joml.Vector2f;
-import org.joml.Vector4f;
+import org.joml.*;
```

Comment:
http://www.javadude.com/posts/20040522-import-on-demand-is-evil/

Reply:
Whoops, I had totally missed that. Thanks @imagejan, will fix 👍

Reply:
fixed